packages/app/src/compiler/index.ts

```diff
@@ -7,7 +7,6 @@ export async function compile(pathname: string, tsconfig: string): Promise<Pipel
   const project = new Project({ tsConfigFilePath: tsconfig });
   const script = project.getSourceFileOrThrow(pathname);
   // TODO: needs to verify if this imported the learnable.
-  const app = script.getImportDeclarationOrThrow('@pipcook/app');
```

Comment:
Sorry, what I did mean is to remove the constant `app`, and we need calling `getImportDeclarationOrThrow` to ensure the input source importing `@pipcook/app`.